src/core/mapping/core_mapper.cc

```diff
@@ -374,6 +378,10 @@ void CoreMapper::map_future_map_reduction(const MapperContext ctx,
                                           FutureMapReductionOutput& output)
 {
```

Comment:
We probably don't want to do this in the case of `LEGATE_CORE_JOIN_EXCEPTION_OP` (in that case we probably want to let Legion do the default behavior, of placing a single value on system memory).

Reply:
https://github.com/nv-legate/legate.core/pull/353/commits/72bc605e5130966e586739c4907a4707a12f608d